CodeChallenge.JobLogger/JobLogger.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+public class JobLogger
+{
+
+    private static bool _logToFile;
+    private static bool _logToConsole;
+    private static bool _logMessage;
+    private static bool _logWarning;
+    private static bool _logError;
+    private static bool LogToDatabase;
+
+    private bool _initialized;
+
+    public JobLogger(bool logToFile, bool logToConsole, bool logToDatabase, bool
+logMessage, bool logWarning, bool logError)
+    {
+        _logError = logError;
+        _logMessage = logMessage;
+        _logWarning = logWarning;
+        LogToDatabase = logToDatabase;
+        _logToFile = logToFile;
+        _logToConsole = logToConsole;
+    }
+
+    public static void LogMessage(string message, bool message, bool warning, bool error)
+    {
+        message.Trim();
```

Comment:
The sanitizing process should be concern of the component that calls this method.